resources/bigtop-1.1.0/BIGTOP-2490.patch

```diff
@@ -24,97 +22,36 @@ index f5a2d64..0fd0603 100644
        $master_port = 7077,
        $worker_port = 7078,
        $master_ui_port = 8080,
-@@ -158,13 +159,24 @@ class spark {
+@@ -158,6 +159,13 @@ class spark {
        ensure => latest,
      }
  
 +    if $zookeeper_connection_string {
 +      $spark_daemon_java_opts = "\"-Dspark.deploy.recoveryMode=ZOOKEEPER -Dspark.deploy.zookeeper.url=${zookeeper_connection_string}\""
-+      $spark_env = "spark/spark-env.ha.sh"
-+      $spark_defaults = "spark/spark-defaults.ha.conf"
 +    }
 +    else {
 +      $spark_daemon_java_opts = "\"-Dspark.deploy.recoveryMode=NONE\""
-+      $spark_env = "spark/spark-env.sh"
-+      $spark_defaults = "spark/spark-defaults.conf"
 +    }
 +
```

Comment:
the above block looks funny to me.. why are we removing the `spark_env` and `spark_defaults` lines here?  And why aren't we including them in the BIGTOP-2490 branch (https://github.com/juju-solutions/bigtop/commit/404d45ca7974f093cc3bfc5dde17c33483c886a5#diff-893948be6099f77b03b5658cb1fcb21fR162)?